### PR TITLE
Feature/functional create new project

### PR DESCRIPTION
**Pull request by VladKalachev**
- Add logger for Node (pino)
- Add server build create-react-app npm/yarn
- Adding db project after building create-react-app
- Fix timeout webpack proxy

### CODE REVIEW

File: packages/cli-ui/server.dev.js [hunk 1]
```diff
@@ -41,5 +47,6 @@ if (process.env.DEV_SERVER.trim() === 'true') {
   })
 }
 
-app.listen(port)
-console.log(`\n Server started on port ${port} \n`)
+app.listen(PORT, () => {
+  logger.info('Server running on port %d', PORT);
+})
```
Comment (KovaSs) [inline packages/cli-ui/server.dev.js:52]: Fix - add new line plz

File: packages/cli-ui/src/pages/CreateProject/CreateProject.tsx [hunk 2]
```diff
@@ -30,12 +34,32 @@ export default function CreateProject () {
...
+      path: selectedPath,
+      manager: manager.value,
+      preset: preset.value
+    }).then((res) => {
+      console.log('res', res)
+      setLoading(false)
+      history.push(Routes.DASHBOARD)
+    }).catch((error) => {
+      console.log('error', error)
+      setLoading(false)
```
Comment (KovaSs) [inline packages/cli-ui/src/pages/CreateProject/CreateProject.tsx:57]: Can add a notifier to the error?

Comment (VladKalachev) [reply packages/cli-ui/src/pages/CreateProject/CreateProject.tsx:57]: yes

File: packages/cli-ui/src/pages/CreateProject/CreateProject.tsx [hunk 3]
```diff
@@ -30,12 +35,35 @@ export default function CreateProject () {
...
+    }).then(() => {
+      setLoading(false)
+      history.push(Routes.DASHBOARD)
+    }).catch((error) => {
+      console.log('error', error)
+      setLoading(false)
+      notification.error({
+        title: error.message,
+        message: error.error.path
+      })
```
Comment (VladKalachev) [inline packages/cli-ui/src/pages/CreateProject/CreateProject.tsx:61]: add notification error

**Pull request merged**